tools/internal_ci/linux/grpc_psm_performance_gke_experiment.sh

```diff
@@ -0,0 +1,148 @@
+# Prebuilt workers for core languages are always built from grpc/grpc.
+if [[ "${KOKORO_GITHUB_COMMIT_URL%/*}" == "https://github.com/grpc/grpc/commit" ]]; then
+    GRPC_CORE_GITREF="${KOKORO_GIT_COMMIT}"
+else
+    GRPC_CORE_GITREF="$(git ls-remote https://github.com/grpc/grpc.git master | cut -f1)"
+fi
+GRPC_JAVA_GITREF="$(git ls-remote https://github.com/grpc/grpc-java.git master | cut -f1)"
+# Kokoro jobs run on dedicated pools.
+DRIVER_POOL=drivers-ci
+WORKER_POOL_8CORE=workers-c2-8core-ci
+# c2-standard-30 is the closest machine spec to 32 core there is
+WORKER_POOL_32CORE=workers-c2-30core-ci
+# Prefix for log URLs in cnsviewer.
+LOG_URL_PREFIX="http://cnsviewer/placer/prod/home/kokoro-dedicated/build_artifacts/${KOKORO_BUILD_ARTIFACTS_SUBDIR}/github/grpc/"
+
+# Update go version.
+TEST_INFRA_GOVERSION=go1.17.1
+go get "golang.org/dl/${TEST_INFRA_GOVERSION}"
+"${TEST_INFRA_GOVERSION}" download
+
+# Clone test-infra repository and build all tools.
+pushd ..
+git clone https://github.com/grpc/test-infra.git
+cd test-infra
+# Tools are built from HEAD.
+git checkout --detach
+make GOCMD="${TEST_INFRA_GOVERSION}" all-tools
+
+# Find latest release tag of test-infra.
+git fetch --all --tags
+TEST_INFRA_VERSION=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+popd
+
+# PSM tests related ENV
+PSM_PROXIED_TEST=proxied
+PSM_PROXYLESS_TEST=proxyless
+PSM_IMAGE_PREFIX=gcr.io/grpc-testing/e2etest/runtime
+PSM_IMAGE_TAG=${TEST_INFRA_VERSION}
+
+# Build test configurations.
+buildConfigs() {
+    local -r pool="$1"
+    local -r table="$2"
+    local -r uniquifier="$3"
+
+    shift 3
+    tools/run_tests/performance/loadtest_config.py "$@" \
+        -t ./tools/run_tests/performance/templates/loadtest_template_psm_"${uniquifier}"_prebuilt_all_languages.yaml \
+        -s driver_pool="${DRIVER_POOL}" -s driver_image= \
+        -s client_pool="${pool}" -s server_pool="${pool}" \
+        -s big_query_table="${table}" -s timeout_seconds=900 \
+        -s prebuilt_image_prefix="${PREBUILT_IMAGE_PREFIX}" \
+        -s prebuilt_image_tag="${UNIQUE_IDENTIFIER}" \
+        -s psm_image_prefix="${PSM_IMAGE_PREFIX}" \
+        -s psm_image_tag="${PSM_IMAGE_TAG}" \
+        -a ci_buildNumber="${KOKORO_BUILD_NUMBER}" \
+        -a ci_buildUrl="${CLOUD_LOGGING_URL}" \
+        -a ci_jobName="${KOKORO_JOB_NAME}" \
+        -a ci_gitCommit="${GRPC_GITREF}" \
+        -a ci_gitCommit_java="${GRPC_JAVA_GITREF}" \
+        -a ci_gitActualCommit="${KOKORO_GIT_COMMIT}" \
+        --prefix="${LOAD_TEST_PREFIX}" -u "${UNIQUE_IDENTIFIER}" -u "${pool}" -u "${uniquifier}"\
+        -a pool="${pool}" \
+        --category=psm \
+        --allow_client_language=c++ --allow_server_language=c++ \
+        -o "psm_${uniquifier}_loadtest_with_prebuilt_workers_${pool}.yaml"
+}
+
+buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "${PSM_PROXIED_TEST}"  --client_channels=8 --server_threads=16 --offered_loads 5000 6000 -l c++ -l java -l python
```

Comment:
You should add a flag here to add an annotation to be used for queueing in the runner, say --annotation-queue=queue -a queue="${WORKERS_8CORE}-proxied", and the same for proxyless.